doc/data/messages/e/exec-used/good.py

```diff
@@ -1 +1,4 @@
-# This is a placeholder for correct code for this message.
+programs = {'do_something': lambda: print("Do something")}
+program = input('Enter a program code to be used: ')
+if programs.get(program):
+    programs[program]()
```

Comment:
I don't understand the good example. Shouldn't user inputs be one of the reason to use exec ?

```suggestion
program = input('Enter code to be executed: ')
exec(program)  # pylint: disable=exec-used
```

Reply:
I'm not so sure about them, these good/bad examples come from `pylint-errors <https://vald-phoenix.github.io/pylint-errors/plerr/errors/basic/W0122>`_.

Reply:
Ok, let's ditch pylint-errors example then. In the suggestion I made I wanted to show that calling code directly is better, but maybe it's a bit naive, if we have to create a function it's clearer that dynamic code creation is not ideal performance wise. Let me update:

```suggestion
def get_user_code(username):
    return input(f'Enter code to be executed please, {username}: ')

username = "Ada"
exec(get_user_code(username ))  # pylint: disable=exec-used
```